src/encoded/static/components/app.js

```diff
@@ -1332,4 +1333,4 @@ module.exports.getRenderedProps = function getRenderedProps(document) {
         props[elem.getAttribute('data-prop-name')] = value;
     }
     return props;
-};
+};
```

Comment:
Seems like this one really is missing the final carriage return.

Reply:
Fixed. I do not recall modifying this.